src/web/app/components/rich-text-editor/rich-text-editor.component.ts

```diff
@@ -34,6 +34,9 @@ export class RichTextEditorComponent implements OnInit {
 
   ngOnInit(): void {
     this.init = this.getEditorSettings();
+    if (window.innerWidth < 768) {
```

Comment:
@wkurniawan07 this should address the issue? It simply hides the toolbar on init. Do you think screen resizing should be considered too?

Reply:
> Do you think screen resizing should be considered too?

Yes, it needs to be.